launch/map4_localization_launch/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>map4_localization_launch</name>
+  <version>0.1.0</version>
+  <description>The map4_localization_launch package</description>
+  <maintainer email="yamato.ando@tier4.jp">Yamato Ando</maintainer>
+  <maintainer email="koji.minoda@tier4.jp">Koji Minoda</maintainer>
+  <maintainer email="kento.yabuuchi.2@tier4.jp">Kento Yabuuchi</maintainer>
+  <maintainer email="ryu.yamamoto@tier4.jp">Ryu Yamamoto</maintainer>
+  <maintainer email="ryohei.sasaki@map.jp">Ryohei Sasaki</maintainer>
+  <license>Apache License 2.0</license>
+  <author email="ryohei.sasaki@map.jp">Ryohei Sasaki</author>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+
+  <build_depend>autoware_cmake</build_depend>
+
+  <exec_depend>automatic_pose_initializer</exec_depend>
+  <exec_depend>ekf_localizer</exec_depend>
+  <exec_depend>gyro_odometer</exec_depend>
+  <exec_depend>ndt_scan_matcher</exec_depend>
+  <exec_depend>pointcloud_preprocessor</exec_depend>
+  <exec_depend>pose_initializer</exec_depend>
+  <exec_depend>topic_tools</exec_depend>
+
```

Comment:
Please list the packages that depend on map4_localization_launch correctly.

Reply:
@YamatoAndo Fixed.